### PR TITLE
Validations: descriptions

### PR DESCRIPTION
**Pull request by SueADarby**

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-smoothstreamingcacheproperty-field-microsoft-web-media-smoothstreaming_1.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingMediaElement.SmoothStreamingCacheProperty Field (Microsoft.Web.Media.SmoothStreaming)
+description: Syntax for the SmoothStreamingMediaElement.SmoothStreamingCacheProperty field it's a dependency that works with the SmoothStreamingCache property.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-smoothstreamingcacheproperty-field-microsoft-web-media-smoothstreaming_1.md:3]: Rewrite this.

**Comment by Rick-Anderson** [comment]
> This PR includes only build validation fixes 

Why are these changed? I don't see any impact in warnings. I'm inclined to reject this PR.

**Comment by SueADarby** [comment]
@Rick-Anderson The changes in this PR were to add missing metadata descriptions that were not originally present. Is there an issue preventing approval and merge for this PR?

**Comment by Rick-Anderson** [comment]
@SueADarby  I don't see one added description. You just changed the descriptions.

**Pull request closed without merging**